### ISSUE TITLE
Allow passing text as StringRes

### ISSUE THREAD

**Issue by mueller-ma** [GrenderG/Toasty#87]
Current:
````
Toasty.info(context, context.getString(R.string.foo), Toast.LENGTH_SHORT).show();
````

Request:
````
Toasty.info(context, R.string.foo, Toast.LENGTH_SHORT).show();
````

**Comment by jpvs0101** [comment]
Yep, that would be super handy...

On Monday, October 15, 2018, mueller-ma <notifications@github.com> wrote:
> Current:
>
> Toasty.info(context, context.getString(R.string.foo),
Toast.LENGTH_SHORT).show();
>
> Request:
>
> Toasty.info(context, R.string.foo, Toast.LENGTH_SHORT).show();
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.<
https://ci5.googleusercontent.com/proxy/GFZyIJXBZ-ScqzQxGOcHCEyhM1G254H6880rMiqdDFmL65ob1ogU1Xjvh9VoMS_J122wMTxQ1VbqeD3DYOsducx7MZ1S8qPPftTC9hFD6-_ZkbgPs2tf52zoapNRYJgjZ33F7kPZ58Nf0OnAEBS4CT_sHfOQwA=s0-d-e1-ft#https://github.com/notifications/beacon/AkhD6qODl8sFbyW94iEWfp87MK-XZ0fuks5ulL7YgaJpZM4Xcvv0.gif>

**Comment by sami286** [comment]
I think that kind of thing is already supported...

**Comment by mueller-ma** [comment]
Right, my issue is a duplicate of https://github.com/GrenderG/Toasty/issues/67, however the change isn't released yet.

**Comment by GrenderG** [comment]
Sorry about this guys, I have just released a new version (`1.3.1`) supporting that feature.

**Comment by mueller-ma** [comment]
Thanks!
